client/hotrod-client/src/main/java/org/infinispan/client/hotrod/jmx/RemoteCacheClientStatisticsMXBean.java

```diff
@@ -0,0 +1,54 @@
+package org.infinispan.client.hotrod.jmx;
+
+public interface RemoteCacheClientStatisticsMXBean {
+
+   /**
+    * Returns the number of remote cache gets
+    */
+   long getRemoteGets();
+
+   /**
+    * Returns the number of remote cache hits
+    */
+   long getRemoteHits();
+
+   /**
+    * Returns the number of remote cache misses
+    */
+   long getRemoteMisses();
+
+   /**
+    * Returns the number of remote cache stores (put, replace)
```

Comment:
"... that were applied - e.g. failed conditional operation does not count. Put counts all the time, though, even if it replaced an equal value."

Reply:
The previous comment was not addressed yet.